### PR TITLE
about pages changes

### PR DESCRIPTION
**Pull request by andreaspavlou**
These are changes to the about pages to see if they are ok and to
integrate before doing other files

### CODE REVIEW

File: lib/views/help/about.html.erb [hunk 1]
```diff
@@ -8,39 +8,37 @@
...
+	  <h2 id="how">Why create AsktheEU.org?</h2>
+	  <p>The European Union is an integral part of national European governance structures, and increasingly influential in every aspect of our daily lives; it has been estimated that up to 50% of national laws originate from Brussels, with the figure rising to 70% when it comes to economic policies.</p>
+	  <p>In a healthy democracy the public knows what its government is doing.</p>
+	  <p>Some information is available on EU websites and in public registers, but it is not enough. If the European public wants to hold the EU to account and to participate in decision-making, we need more information. </p>
+	  <p>At present, even though there are over 500 million EU citizens, the EU institutions receive only about 10,000 requests for documents per year.</p>
+	  <p>AsktheEU.org is designed to help members of the public find out more about what the EU is doing and to get access to information of use to you so that you can use it to engage in public debates and contribute to better decisions being taken at the EU level.</p>
 
+	  <h2 id="users">What does Access Info Europe do?</h2>
```
Comment (garethrees) [inline lib/views/help/about.html.erb]: I think this section loses some information about who can actually make requests, particularly the following line:

> Every one of the 500 million people living in the European Union (both citizens and residents) has the legally guaranteed right to ask for documents from EU bodies.

Worth adding that back in?

**Comment by garethrees** [comment]
Looks great in general. One [comment](https://github.com/mysociety/asktheeu-theme/pull/20#discussion_r51430554) that I think we should address, but will merge after we decide on that.

**Comment by andreaspavlou** [comment]
Ah yes, please include in 'Why create asktheEU?', in place of the para that goes 'At present, even though there are over 500 m.....'

> Every one of the 500 million people living in the European Union (both citizens and residents) has the legally guaranteed right to ask for documents from EU bodies - but EU institutions only receive around 10,000 requests per year.

Thanks!

**Comment by garethrees** [comment]
Great, will get that merged today/tomorrow. Feel free to open up new pull requests for the other pages :+1:

**Comment by andreaspavlou** [comment]
Hi Gareth,,

I think I just did a pull request for the EU officials text, but not 
sure if I did it properly. :-[



-- 

*Andreas Pavlou*
RTI Campaigner & Researcher
*Access Info Europe* | http://www.access-info.org
/Twitter: /@andreaspavlou @Access_Info @AsktheEU
/Madrid office/: +34 913 656 558
/Mobile/: +34 664 779 124

On 02/02/2016 12:47, Gareth Rees wrote:
>
> Great, will get that merged today/tomorrow. Feel free to open up new 
> pull requests for the other pages :+1:
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mysociety/asktheeu-theme/pull/20#issuecomment-178524888>.
>



---
El software de antivirus Avast ha analizado este correo electrónico en busca de virus.
https://www.avast.com/antivirus

**Comment by garethrees** [comment]
Looks like you've re-opened this pull request with a new commit. No problem for this page, I'll just sort it out manually. How are you making edits to the pages?

**Comment by andreaspavlou** [comment]
I am using a desktop interface that Louise showed me, but I am not sure if I have used it right this time... I change the files which have been downloaded onto my desktop.

**Comment by garethrees** [comment]
Ah cool. I'm not super familiar with GitHub Desktop (assuming that's what you're using) but I think you want to follow this guide https://help.github.com/desktop/guides/contributing/sending-a-pull-request/. You need to [sync your branch](https://help.github.com/desktop/guides/contributing/syncing-your-branch/) from master so that you get the already merged changes.

**Comment by garethrees** [comment]
Merged as b733189025d93c373be9fb12d4b0ac80014097ce

**Pull request closed without merging**